pkg/sql/logictest/testdata/logic_test/row_level_ttl

```diff
@@ -442,6 +450,217 @@ CREATE TABLE public.tbl (
+                                                                                          FAMILY fam_0_id_expire_at (id, expire_at)
+) WITH (ttl = 'on', ttl_expiration_expression = '(expire_at)', ttl_job_cron = '@hourly')
+
+statement ok
+ALTER TABLE tbl_create_table_ttl_expiration_expression RESET (ttl)
+
+subtest end
+
+subtest alter_table_ttl_expiration_expression
+
+statement ok
+CREATE TABLE tbl_alter_table_ttl_expiration_expression (
+  id INT PRIMARY KEY,
+  expire_at TIMESTAMP
+)
+
+statement ok
+ALTER TABLE tbl_alter_table_ttl_expiration_expression SET (ttl_expiration_expression = 'expire_at')
+
+query T
+SELECT create_statement FROM [SHOW CREATE TABLE tbl_alter_table_ttl_expiration_expression]
+----
+CREATE TABLE public.tbl_alter_table_ttl_expiration_expression (
+                                                                                          id INT8 NOT NULL,
+                                                                                          expire_at TIMESTAMP NULL,
+                                                                                          CONSTRAINT tbl_alter_table_ttl_expiration_expression_pkey PRIMARY KEY (id ASC),
+                                                                                          FAMILY fam_0_id (id),
+                                                                                          FAMILY fam_1_expire_at (expire_at)
+) WITH (ttl = 'on', ttl_expiration_expression = '(expire_at)', ttl_job_cron = '@hourly')
+
+# try setting it again
+statement ok
+ALTER TABLE tbl_alter_table_ttl_expiration_expression SET (ttl_expiration_expression = 'expire_at + ''5 minutes'':::INTERVAL')
+
+query T
+SELECT create_statement FROM [SHOW CREATE TABLE tbl_alter_table_ttl_expiration_expression]
+----
+CREATE TABLE public.tbl_alter_table_ttl_expiration_expression (
+                                                                                                                   id INT8 NOT NULL,
+                                                                                                                   expire_at TIMESTAMP NULL,
+                                                                                                                   CONSTRAINT tbl_alter_table_ttl_expiration_expression_pkey PRIMARY KEY (id ASC),
+                                                                                                                   FAMILY fam_0_id (id),
+                                                                                                                   FAMILY fam_1_expire_at (expire_at)
+) WITH (ttl = 'on', ttl_expiration_expression = '(expire_at + '5 minutes':::INTERVAL)', ttl_job_cron = '@hourly')
+
+statement ok
+ALTER TABLE tbl_alter_table_ttl_expiration_expression RESET (ttl)
+
+subtest end
+
+subtest create
```

Comment:
hmm...this `subtest create` seems unnecessary?

(i'm not a big fan of subtests cause the context can get easily lost)

Reply:
Yeah this was a mistake.  

Is there another way of breaking up large test files into test cases? Convert each test file into a directory with a file per test case? 

Without being able to break it up, it is hard to tell what is actually being tested.